### PR TITLE
[improvement] show dialog on resume click

### PR DESCRIPTION
**Pull request by jadehjk**
On clicking the resume button, it now shows a dialog with the contents of the pdf

### CODE REVIEW

File: src/Client/scripts/App.tsx [hunk 1]
```diff
@@ -43,6 +46,13 @@ const App: React.FC<{}> = () => {
       'Projects': projects
    };
 
+  
+   const [resumeOpen, setResumeOpen] = useState(false);
```
Comment (EunPyoLee) [inline src/Client/scripts/App.tsx:50]: ```const [resumeOpen, setResumeOpen] = React.useState<boolean>(null);``` for type

File: src/Client/Styles/Core/_resume.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@import '../vars.scss';
+
+.resume {
+    .resume-dialog {
```
Comment (EunPyoLee) [inline src/Client/Styles/Core/_resume.scss:4]: Give one line space above the inner definition for better readability
`.resume{
           (space)
                .resume-dialog{...`

File: src/Client/scripts/App.tsx (hunk 1 above)
Comment (EunPyoLee) [reply src/Client/scripts/App.tsx:50]: And move this code to line 34 (to be defined at the place where similar thing defined)

File: src/Client/Styles/Core/_resume.scss (hunk 2 above)
Comment (jadehjk) [reply src/Client/Styles/Core/_resume.scss:5]: fixed!

File: src/Client/scripts/App.tsx (hunk 1 above)
Comment (jadehjk) [reply src/Client/scripts/App.tsx]: fixed!

**Pull request merged**